packages/cli/src/commands/codegen.ts

```diff
@@ -30,8 +30,10 @@ export class CodeGen extends Command {
 
       if (hasPrisma && hasPrismaSchema) {
         let prismaSpinner = log.spinner(`Generating Prisma client`).start()
-        void runPrisma(["generate"], true).then((success) =>
-          success ? prismaSpinner.succeed() : prismaSpinner.fail(),
+        void runPrisma(["generate"], true).then((result) =>
+          result.success
+            ? prismaSpinner.succeed()
+            : prismaSpinner.fail() && console.log("\n" + result.stderr)
```

Comment:
It works okay, but I think we can slightly improve it — since there was an error, we should exit with code 1. What about this:

```
          if (result.success) {
            prismaSpinner.succeed()
          } else {
            prismaSpinner.fail()
            console.log("\n" + result.stderr)
            process.exit(1)
          }
```